easybuild/easyconfigs/g/gtest/gtest-1.7.0-CrayGNU-2015.11.eb

```diff
@@ -1,4 +1,4 @@
-easyblock = "Tarball"
+easyblock = "CMakeMake"
```

Comment:
@migueldiascosta what's going on here? this is way beyond just fixing source URLs?

Reply:
@migueldiascosta re-ping?

Reply:
@boegel yeah, the upstream package seems to have changed (do you have an old copy to compare?). Since changing the easyconfig would mean it would fail for those who have a downloaded copy of the old file, I suppose it's better to close this PR.